showcase-app/test-cases.md

```diff
@@ -95,3 +95,32 @@ If you want to run the tests interactively, you use `npm run test:ui`.
   - Reload the page.
   - Verify that the active profile is still displayed.
 
+### 11. Login without extension using WebID
+
+- **Description**: Test the login flow using a WebID input field.
+- **Test Steps**:
+  - Register a profile.
+  - Load main page.
+  - Enter the profile's WebID in the input field, just like a user would.
+  - Log in using profile's credentials.
+  - Verify that upon redirect to the main page, the app is authenticated.
+
+### 12. Displays logged in profile's WebID when available
+
+- **Description**: Test if the WebID's name is displayed when it's returned from the identity provider.
```

Comment:
The description still mentions the IDP.

Reply:
I haven't change the code to parse the name from the WebID. Maybe there is a bug there.
However, when I check your WebID I get a regular website. When I check the WebID I generate from the solidlab playground I get an endpoint that returns text/turtle response:
https://pod.playground.solidlab.be/serge2/profile/card#me

Reply:
My server does content negotiation. If you ask for Turtle you will get Turtle:

```shell
curl -L -H "accept: text/turtle" https://pieterheyvaert.com/#me
```

Reply:
thx. the problem is that your webid's name is specified for a locale (even 2 locales in your case).
in the inrupt library there isn't a method to get the name irrespective of locale: it's either:
- give me a string (the name) for locale X https://docs.inrupt.com/developer-tools/api/javascript/solid-client/modules/thing_get.html#getstringwithlocale
- give me a string without locale (ignores names with any locale) https://docs.inrupt.com/developer-tools/api/javascript/solid-client/modules/thing_get.html#getstringnolocale

https://docs.inrupt.com/developer-tools/api/javascript/solid-client/modules/thing_get.html

How should I adjust the code?
Do you know of a method to find out what locales are available in the webid?

Reply:
@pheyvaer never mind, I found it: https://docs.inrupt.com/developer-tools/api/javascript/solid-client/modules/thing_get.html#getstringbylocaleall